src/components/Tasks/AddNewTaskItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useContext } from 'react';
```

Comment:
See the rest of the code, we follow a different convention.
We do `import * as React from 'react'` and then use `React.useState` in the code.